.github/dependabot.yml

```diff
@@ -68,7 +76,7 @@ updates:
       time: "04:00"
       timezone: "Europe/Amsterdam"
     reviewers:
-    - "infonl/dimpact"
+      - "infonl/dimpact"
```

Comment:
Is this formatting change correct? No Spotless issue?

Reply:
or maybe this is more correct? as long as it still works on GitHub :-)

Reply:
It is now aligning with 3 other identical reviewers lines in this yaml; so quite sure assuming it works is quite safe.